static/docs/command-reference/update.md

```diff
@@ -39,6 +39,12 @@ to learn how to "update" fixed-revision imports.
 
 ## Options
 
+- `--rev` - specific
+  [Git revision](https://git-scm.com/book/en/v2/Git-Internals-Git-References)
```

Comment:
No need to link "Git revision" here since it's linked in the paragraph above. For some reason we don't use links or `<abbr>` tooltips in the **Options** sections, so let's not include it here for consistency.

Reply:
Actually, never mind about this link. I see we have this in https://dvc.org/doc/command-reference/import#options. Resolving.